### PR TITLE
CON-1050: Update doc for Netskope REST APIv2

### PR DESCRIPTION
**Pull request by duchatran**
## Purpose of this pull request

This pull request (PR) ...

Issue number: https://jasklabs.atlassian.net/browse/CON-1330

## Select the type of change:
- [ ] Minor Changes - Typos, formatting, slight revisions
- [] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md [hunk 1]
```diff
@@ -46,14 +46,27 @@ Hover your mouse over the status icon to view a tooltip with details on the dete
...
+### Netskope REST API v1 (Deprecated, for Sumo Logic Netskope source v1.3.1 or lower)
+Netskope RESTv1 APIs use an auth token to make authorized calls to the
 API. This section demonstrates how to obtain a token from the Netskope
 user interface (UI).
 
 To obtain a Netskope auth token, do the following:
 
 1. Login to Netskope as the Tenant Admin.
-1. Go to the API portion of the Netskope, **Settings \> Tools \> Rest API**.
+1. Go to the API portion of the Netskope, **Settings \> Tools \> Rest API v1**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md:69]:
```suggestion
1. Go to the API portion of the Netskope, **Settings** > **Tools** > **Rest API v1**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md [hunk 2]
```diff
@@ -46,14 +46,27 @@ Hover your mouse over the status icon to view a tooltip with details on the dete
...
+### Netskope REST API v2 
+
+Netskope REST APIv2 provides an easy way to extend the Netskope platform to build to use-cases specific to your organization. Endpoints cover key areas such as Events, Alerts, Reports, Clients and more. 
+
+To obtain a Netskope REST API v2 auth token, do the following:
+
+1. Login to Netskope as the Tenant Admin.
+1. Go to the API portion of the Netskope, **Settings \> Tools \> Rest API v2**.
+1. Click "New Token", provide the token name, expiration duration and add the following endpoints with READ privilege:
+/api/v2/events/dataexport/events/alert, /api/v2/events/dataexport/events/page, /api/v2/events/dataexport/events/infrastructure, /api/v2/events/dataexport/events/application, /api/v2/events/dataexport/events/network,/api/v2/events/dataexport/events/alert/audit and /api/v2/events/data/alert
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md:58]:
```suggestion
1. Log in to Netskope as the Tenant Admin.
1. Go to the API portion of the Netskope, **Settings** > **Tools** > **Rest API v2**.
1. Click "New Token", provide the token name and expiration duration, then add the following endpoints with READ privilege:
  ```
  /api/v2/events/dataexport/events/alert
  /api/v2/events/dataexport/events/page 
  /api/v2/events/dataexport/events/infrastructure
  /api/v2/events/dataexport/events/application
  /api/v2/events/dataexport/events/network
  /api/v2/events/dataexport/events/alert/audit
  /api/v2/events/data/alert 
  ```
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md [hunk 3]
```diff
@@ -46,14 +46,26 @@ Hover your mouse over the status icon to view a tooltip with details on the dete
...
-Netskope REST APIs use an auth token to make authorized calls to the
+### Netskope REST API v2 
+
+Netskope REST APIv2 provides an easy way to extend the Netskope platform to build to use-cases specific to your organization. Endpoints cover key areas such as Events, Alerts, Reports, Clients and more. 
+
+To obtain a Netskope REST API v2 auth token, do the following:
+
+1. Log in to Netskope as the Tenant Admin.
+1. Go to the API portion of the Netskope, **Settings** > **Tools** > **Rest API v2**.
+1. Click "New Token", provide the token name and expiration duration, then add the following endpoints with READ privilege:
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md:57]: 1. Log in to Netskope as the Tenant Admin.
1. Go to the API portion of the Netskope, **Settings** > **Tools** > **Rest API v2**.
1. Click "New Token", provide the token name and expiration duration, then add the following endpoints with READ privilege: `/api/v2/events/dataexport/events/alert`, `/api/v2/events/dataexport/events/page`, `/api/v2/events/dataexport/events/infrastructure`, `/api/v2/events/dataexport/events/application`, `/api/v2/events/dataexport/events/network`, `/api/v2/events/dataexport/events/alert/audit` and `/api/v2/events/data/alert`

Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md:57]:
```suggestion
1. Log in to Netskope as the Tenant Admin.
1. Go to the API portion of the Netskope, **Settings** > **Tools** > **Rest API v2**.
1. Click "New Token", provide the token name and expiration duration, then add the following endpoints with READ privilege: `/api/v2/events/dataexport/events/alert`, `/api/v2/events/dataexport/events/page`, `/api/v2/events/dataexport/events/infrastructure`, `/api/v2/events/dataexport/events/application`, `/api/v2/events/dataexport/events/network`, `/api/v2/events/dataexport/events/alert/audit` and `/api/v2/events/data/alert`
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md [hunk 4]
```diff
@@ -46,14 +46,26 @@ Hover your mouse over the status icon to view a tooltip with details on the dete
...
+
+1. Log in to Netskope as the Tenant Admin.
+1. Go to the API portion of the Netskope, **Settings** > **Tools** > **Rest API v2**.
+1. Click "New Token", provide the token name and expiration duration, then add the following endpoints with READ privilege: `/api/v2/events/dataexport/events/alert`, `/api/v2/events/dataexport/events/page`, `/api/v2/events/dataexport/events/infrastructure`, `/api/v2/events/dataexport/events/application`, `/api/v2/events/dataexport/events/network`, `/api/v2/events/dataexport/events/alert/audit`, and `/api/v2/events/data/alert`
+1. Copy the token in the next dialog box and save it somewhere as it won't be visible after.
+
+### Netskope REST API v1 (Deprecated, for Sumo Logic Netskope source v1.3.1 or lower)
+Netskope RESTv1 APIs use an auth token to make authorized calls to the
 API. This section demonstrates how to obtain a token from the Netskope
 user interface (UI).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/netskope-source.md:63]:
```suggestion
### Netskope REST API v1 (Deprecated)

:::caution Deprecated
For Sumo Logic Netskope source v1.3.1 or lower.
:::

Netskope RESTv1 APIs use an auth token to make authorized calls to the API. This section demonstrates how to obtain a token from the Netskope user interface (UI).
```

**Review by kimsauce — DISMISSED** [review]
one suggestion (see comments)

**Review by JainM6 — APPROVED** [review]
LTGM

**Review by JainM6 — APPROVED** [review]

**Comment by duchatran** [comment]
@JainM6 I did sign the CLA yesterday (using my sumo email) - just signed again using your link.

**Pull request merged**